### PR TITLE
resolve minor TODOs

### PR DESCRIPTION
**Pull request by nklaassen**

### CODE REVIEW

File: lib/auth/bot.go [hunk 1]
```diff
@@ -288,8 +288,6 @@ func (a *Server) updateBotInstance(
 		}
 	}
 
-	// TODO(nklaassen): consider recording both public keys once they are
-	// actually separated.
```
Comment (nklaassen) [inline lib/auth/bot.go:292]: tbot always uses the same key for SSH and TLS when it requests both

File: lib/auth/init.go [hunk 2]
```diff
@@ -1240,7 +1240,6 @@ func checkResourceConsistency(ctx context.Context, keyStore *keystore.Manager, c
 
 // GenerateIdentity generates identity for the auth server
 func GenerateIdentity(a *Server, id state.IdentityID, additionalPrincipals, dnsNames []string) (*state.Identity, error) {
-	// TODO(nklaassen): split SSH and TLS keys for host identities.
```
Comment (nklaassen) [inline lib/auth/init.go:1243]: not splitting SSH/TLS keys for host identities

File: lib/cryptosuites/suites.go [hunk 3]
```diff
@@ -100,7 +100,6 @@ const (
 	// HostSSH represents a host SSH key.
 	HostSSH
 	// HostIdentity represents a key used for a Teleport host identity.
-	// TODO(nklaassen): split SSH and TLS keys used for host identities.
```
Comment (nklaassen) [inline lib/cryptosuites/suites.go:103]: not splitting SSH/TLS keys for host identities

File: lib/reversetunnel/cache.go [hunk 4]
```diff
@@ -113,8 +113,6 @@ func (c *certificateCache) generateHostCert(ctx context.Context, principals []st
 		return nil, trace.Wrap(err)
 	}
 
-	// TODO(nklaassen): request only an SSH cert, we don't need TLS here.
-	// GenerateHostCert needs support for this.
```
Comment (nklaassen) [inline lib/reversetunnel/cache.go:117]: it already does this idk why i wrote this TODO

**Review by rosstimothy — APPROVED** [review]

**Review by marcoandredinis — APPROVED** [review]

**Pull request merged**